server/src/main/java/org/jboss/as/server/parsing/StandaloneXml_4.java

```diff
@@ -111,19 +110,22 @@
 class StandaloneXml_4 extends CommonXml implements ManagementXmlDelegate {
 
     private final AccessControlXml accessControlXml;
+    private final ParsingOptions parsingOptions;
     private AuditLogXml auditLogDelegate;
-
-    private final ExtensionXml extensionXml;
-    private final ExtensionRegistry extensionRegistry;
     private final Namespace namespace;
+    private ExtensionHandler extensionHandler;
+
+    public interface ParsingOptions{
+        boolean strict();
```

Comment:
I've been making minor comments; this one is the only serious concern I have. This is very vague contract.